docs/advanced/full-node.md

```diff
@@ -0,0 +1,68 @@
+
+## Steps
+
+Below are the basic high-level steps to install a full node on any hardware and OS. These instructions use [dcrinstall](../wallets/cli/cli-installation.md), the automatic installer and upgrader software. In addition to installing `dcrd`, it will also install `dcrwallet` (which allows you to create wallets) and `dcrctl` (which allows you to control `dcrwallet` and `dcrd` from the command line). You can also just install `dcrd` on its own from [binaries or source files](https://github.com/decred/dcrd#installing-and-updating); a full node does not require a wallet unless you want to use your full node to validate and broadcast your own transactions.
+
+1. **Download and install dcrd**
+	- Follow the instructions for your OS in the [CLI Installation guide](https://docs.decred.org/wallets/cli/cli-installation/). The `dcrd` executable will be installed in a directory named `/decred` under your OS's home directory. 
+1. **Start `dcrd`**
+	- Navigate to the `/decred` directory and launch `dcrd`. See the [Operating System Differences](../wallets/cli/os-differences.md) page for OS-specific commands.
+	- `dcrd` will boot up, begin connecting to peers and downloading the full Decred blockchain.
+1. **Enable incoming connections**
+	- `dcrd` will automatically begin downloading the blockchain and connecting to peers. To maximise your positive impact on the Decred network, it is important to allow inbound peers to connect to your node. Allowing inbound peers will allow new participants in the Decred network to connect to your node, and also enable your node to serve lightweight clients such as SPV wallets. This may require changing the settings of your router (to allow outbound connections). If this is the case:
+		1. Find your [Local IP address](https://www.howtogeek.com/236838/how-to-find-any-devices-ip-address-mac-address-and-other-network-connection-details/) and [External IP address](https://whatismyipaddress.com/).
+		1. Edit the settings of your router to open port 9108 (port forwarding). This [process](https://www.wikihow.com/Set-Up-Port-Forwarding-on-a-Router) depends on the type of router you have. 
+	- If running a firewall, you'll also need to configure it to allow inbound connections on port 9108.
+1. **Wait for dcrd to sync**
+	- Check a Decred block explorer such as [dcrdata](https://dcrdata.decred.org/) and wait until the block height displayed matches the block height shown locally.
+1. **Restart dcrd**
+	- Restart `dcrd` and pass it your external IP address using the following flag `--externalip=[your.external.ip]`
+	- If you don't wish to pass your IP address every time you restart `dcrd`, you can put it in dcrd's [configuration file](../wallets/cli/startup-basics.md#configuration-file-locations). \
```

Comment:
Took one last pass over this and noticed a rogue `\` at the end of this line.

The PR is good to merge once this is remove.

Reply:
Removed